game/plugins/src/main/kotlin/gg/rsmod/plugins/content/magic/PawnExt.kt

```diff
@@ -26,10 +26,14 @@ fun Player.canTeleport(type: TeleportType): Boolean {
     return true
 }
 
+fun Pawn.resetCombatActions() {
```

Comment:
```suggestion
fun Pawn.prepareForTeleport() {
```

I think the current name may be misleading to be directly related with combat, but it should be clear that it's usage it specifically for teleporting. Though the suggestion may not be the best name, it'll be a bit more clear that it's for teleporting and not for normal combat purposes

Reply:
Ok, done man 👍